src/misc/ListModel.ts

```diff
@@ -38,6 +39,8 @@ export interface ListModelConfig<ElementType> {
 	loadSingle(elementId: Id): Promise<ElementType | null>
 
 	sortCompare(entity1: ElementType, entity2: ElementType): number
+
+	autoSelectBehavior: (() => ListAutoSelectBehavior) | null
```

Comment:
does it have to be optional?

Reply:
I did not understand, sorry.

To me it's confusing because we define some behavior for the list and if it's not set it's not clear from the outside what the behavior should be.

Is default/null option distinct from other ones we can pass in?